libs/s25main/Game.cpp

```diff
@@ -94,8 +95,9 @@ void Game::RunGF()
 
     if(world_.HasLua())
         world_.GetLua().EventGameFrame(em_->GetCurrentGF());
-    // Update statistic every 750 GFs (30 seconds on 'fast')
-    if(em_->GetCurrentGF() % 750 == 0)
+    // Update statistic every 30 seconds
+    constexpr unsigned GFsIn30s = std::chrono::duration<unsigned>(30) / SPEED_GF_LENGTHS[referenceSpeed];
```

Comment:
Shame here. The `30s` looked nicer, but well... I guess this is because `seconds` uses a 64bit type and hence is wider than `unsigned` while for `minutes` it is a 32 bit type. I'd probably use a cast at the end, but doesn't matter which way.